src/main/java/de/pandemieduell/api/GamesController.java

```diff
@@ -0,0 +1,47 @@
+package de.pandemieduell.api;
+
+import de.pandemieduell.model.GameState;
+import de.pandemieduell.model.Round;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GamesController {
+
+  @RequestMapping(value = "/registerUser", method = RequestMethod.POST, consumes = "text/plain")
+  public long registerUser(@RequestBody String username) {
+    return 0; // TODO implement
+  }
+
+  @RequestMapping(value = "/joinGame", method = RequestMethod.GET)
+  public long joinGame(@RequestParam("playerId") long playerId) {
+    return 0; // TODO implement
+  }
+
+  @RequestMapping(value = "games/{gameId}/currentRound", method = RequestMethod.GET)
+  public Round getCurrentRound(@PathVariable long gameId) {
+    return null; // TODO implement
+  }
+
+  @RequestMapping(value = "games/{gameId}/round/{roundId}", method = RequestMethod.GET)
+  public Round getRound(@PathVariable long gameId, @PathVariable long roundId) {
+    return null; // TODO implement
+  }
+
+  @RequestMapping(value = "games/{gameId}/state", method = RequestMethod.GET)
+  public GameState getCurrentGameState(@PathVariable long gameId) {
```

Comment:
I think it would be easier if this was a `GET /games/{gameId}` and we would get all relevant information about the current game. That means the current turn, which cards can be played, the state of the world, etc.